### ISSUE TITLE
How to Handle Multiple Accounts? i.e. Have a F2P Account

### ISSUE THREAD

**Issue by eversor431** [HearthSim/Hearthstone-Deck-Tracker#2893]
Hi,

I have a F2P account in addition to my regular account. I had a lot of stats and stored matches from the F2P account which got completely wiped when I went to HearthStats and went to sync.

Also, it seems to have uploaded the matches from my regular account only, deleted the F2P account matches, and not uploaded the scores for the matches. So for my Token Yogg deck, I have 9 matches played and a 0-0 win loss record.

It also deleted my most recent 5 matches from my regular account; I should have 5 games played with Midrange Shaman and those are all gone.

What is going on, and how to do I stop it from deleting my matches?

I unchecked delete decks and games automatically; I cannot believe you have this set on by default.

-eversor431

**Comment by riQQ** [comment]
Check this: https://github.com/HearthSim/Hearthstone-Deck-Tracker/wiki/FAQ#no-stats.

**Comment by eversor431** [comment]
I did. Everything is correct.

Everything was going fine until I synced with HearthStats. Then I lost a bunch of games. The local replays of those deleted games are still on my computer.

Also, the constructed stats is now showing some Arena, Brawl, and Friendly games which I do not have checked.

![image](https://cloud.githubusercontent.com/assets/15005423/19414836/c86584c2-9328-11e6-9ad9-993beed68a66.png)


![image](https://cloud.githubusercontent.com/assets/15005423/19414839/da603686-9328-11e6-81d5-edd59179f9be.png)
